test/cli.py

```diff
@@ -304,16 +337,26 @@ def test_004_cli_report(self):
             "nvidia_latency",
             "x86_latency",
         ]
-        linear_summary = summary[0]
+        linear_summary = summary[1]
```

Comment:
fyi @danielholanda this line of code is brittle, it depends on the alphabetical order of the script names in the corpus. Didn't have time to make it more robust.

Reply:
Yeah... that might break some other tests that are in existing PRs and rely on that brittle logic...